client/src/pages/Dashboard/components/ConversationItem.js

```diff
@@ -0,0 +1,62 @@
+import { Avatar, Badge, Grid, Typography } from '@material-ui/core';
+import { makeStyles } from "@material-ui/core/styles";
+import OnlineCircle from '../../../components/OnlineCirclie';
+
+const useStyles = makeStyles(theme => ({
+    selected: {
+        backgroundColor: '#eee'
+    },
+    interactive: {
+        transition: 'all 0.3s ease',
+        borderRadius: 8,
+        boxShadow: "0 2px 10px 0 #5885C40C",
+        
+        '&:hover': {
+            cursor: 'pointer',
+            transform: 'scale(1.03)',
+            backgroundColor: '#5885C40B'
+        }
+    },
+    avatar: {
+        width: 44,
+        height: 44
+    },
+    subText: {
+        color: '#9CADC8',
+        fontSize: 12
+    }
+}));
+
+export default function ConversationItem({ title, detail, onlineStatus, badge=true, selected=false, interactive = true }) {
+    const classes = useStyles();
+
+    return (
+        <Grid className={`${interactive&&!selected ? classes.interactive : null} ${selected ? classes.selected : null}`} container direction="row" alignItems="center" spacing={3}>
```

Comment:
I would've recommended using the [`List` and `ListItem`](https://material-ui.com/components/lists/) components for the lists of users and conversations instead of nesting `Grid`s.

Reply:
Is there any performance related reason for this? It seems like List does not provide spacing between items

Reply:
You can always style them to add spacing. It's more for semantics and readability, and the fact that it provides simple helpers for things like sub-text and adornments.

Reply:
Updated